doc/ale-php.txt

```diff
@@ -169,6 +169,23 @@ g:ale_php_phpstan_configuration               *g:ale_php_phpstan_configuration*
   This variable sets path to phpstan configuration file.
 
 
+===============================================================================
+psalm_lsp                                                   *ale-php-psalm_lsp*
+
+g:ale_php_psalm_lsp_executable                 *g:ale_php_psalm_lsp_executable*
+                                               *b:ale_php_psalm_lsp_executable*
+  Type: |String|
+  Default: `'psalm'`
+
+  This variable sets the executable used for Psalm's language server.
+
+g:ale_php_psalm_lsp_use_global                 *g:ale_php_psalm_lsp_use_global*
+                                               *b:ale_php_psalm_lsp_use_global*
+  Type: |Boolean|
+  Default: `get(g:, 'ale_use_global_executables', 0)`
+
+  This variable force globally installed Psalm language server.
```

Comment:
Nitpick: `s/force/forces/`

Reply:
Fixed both here and the place I blindly copy-pasted from